libs/nest/feature-flags/src/lib/feature-flag.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from '@island.is/nest/config'
+
+export const FeatureFlagConfig = defineConfig({
+  name: 'FeatureFlag',
+  load: (env) => ({
+    sdkKey: env.required(
+      'CONFIGCAT_SDK_KEY',
+      'YcfYCOwBTUeI04mWOWpPdA/KgCHhUk0_k2BdiKMaNh3qA',
```

Comment:
Should the SDK key be in the source?

Reply:
It's already in the codebase and included in all of our production browser bundles. So I'll leave this for now until we decide to remove it globally at some point.